### DISCUSSION TITLE
💫 Sorobanathon draws digital artist into NFTs on Stellar

### DISCUSSION THREAD

**Discussion by mgaylor** [stellar/sorobanathon#168] [General]
I'm thrilled to bring you fresh mints from Oak, a talented young artist launching her first NFT collection just for Sorobanathon. Since discovering her tablet's free drawing app, Oak's fan art has been 🔥

✨ Follow Oak on Insta 
**[@eyrieoak](https://www.instagram.com/eyrieoak/)**

✨ Purchase Oak's debut NFTs on 
**[Litemint](https://litemint.com/explore?n=f8lucx.litemint.store)**

### <a href="https://litemint.com/items/GCISU4CA7V5ULHZFXNSUSAICOPOJG75MCU4AMTFYJOD2ZRK7VJRULGO7/SFL0" target="new">Squatahune Temple</a>
![Prologue: Squaktahune Temple](https://litemint.azureedge.net/userdata/original-SFL0-GCISU4CA7V5ULHZFXNSUSAICOPOJG75MCU4AMTFYJOD2ZRK7VJRULGO7.png)

### <a href="https://litemint.com/items/GASR5PHJQ3PIBE56KPZRDJNPOXHEPC2SDEMYVHT4OYL4DXIK23F27KHV/SFL1" target="new">Hello World
![Chapter 1: Hello World](https://ipfs.io/ipfs/QmYM4oFJi3ZBVFwzZrtW2Ebycb3Nvc9oi3CwJpFsC5z7AZ)</a>
### <a href="https://litemint.com/items/GBAHSMZNRWVBT4RMIQAPEGUWUG5ECKOFXH7TUJ7CBMTXKAPRLPBVVVV7/SFL2" target="new"> Grappling Hook</a>
![Chapter 2: Grappling Hook](https://ipfs.io/ipfs/QmQB1vMEZy48S3JLNRgLCbupuTvb663dW7yAyvNTepPczi)

⭐️ Thanks for supporting digital artists and the creator ecosystem!

**Comment by rawritude** [comment]
Wow these are great. Not quite sure what's going on with litemint

**Reply by mgaylor** [reply]
It took several hours but they eventually showed up.  Thanks for the kind words!

**Reply by JuanConner** [reply]
It's jammed because everyone is frantically finishing up lol

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 900 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Comment by JuanConner** [comment]
Thanks for reaching out on Github, I dropped the images in the comment section, and filled out the form you sent. I'm including a sworn Affidavit of my legal identity just in case there are any discrepancies. Thank you for your time and patience, it was fun being involved with this project.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by stellarjohn** [comment]
Hey! Please resubmit the KYC form. We need the confirmation code to move forward with payment :)

**Reply by JuanConner** [reply]
I did submit the kyc form sir, was there an issue with the one I sent?

**Reply by mgaylor** [reply]
Done!
